include/extractor/guidance/intersection.hpp

```diff
@@ -25,7 +25,14 @@ namespace extractor
 namespace guidance
 {
 
-// the shape of an intersection only knows about edge IDs and bearings
+// the shape of an intersection only knows about edge IDs and bearings. The
+// associated turn is described by an angle and an instruction that is used to announce it.
+// The Turn Operation indicates what is exposed to the outside of the turn analysis.
+//
+// `angle`      is given counter-clockwise:
+//              0 = uturn, 90 = right, 180 = straight, 270 = left
```

Comment:
the intersection shape itself does not know about where you come from. It does not offer an `angle` property.